plugins/generator-1.14.4/forge-1.14.4/templates/block.java.ftl

```diff
@@ -119,13 +127,18 @@ public class ${name}Block extends ${JavaModName}Elements.ModElement {
 					</#if>
 			);
 
-			<#if data.rotationMode == 1 || data.rotationMode == 3>
-			this.setDefaultState(this.stateContainer.getBaseState().with(FACING, Direction.NORTH));
-			<#elseif data.rotationMode == 2 || data.rotationMode == 4>
-			this.setDefaultState(this.stateContainer.getBaseState().with(FACING, Direction.NORTH));
-			<#elseif data.rotationMode == 5>
-			this.setDefaultState(this.stateContainer.getBaseState().with(FACING, Direction.SOUTH));
-        	</#if>
+			this.setDefaultState(this.stateContainer.getBaseState()

```

Comment:
Why is this outside any IF statement?

Reply:
okay I'll add a if statement checking if its |not blockbase 0| or |waterloggable|